tests/bluetooth/bsim_bt/bsim_test_mesh_pst/src/test_pst_prov.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mesh_pst_test.h"
+#include "settings_test_backend.h"
+#include <bluetooth/mesh.h>
+#include <sys/reboot.h>
+#include "mesh/net.h"
+#include "mesh/app_keys.h"
+
+
+#define LOG_MODULE_NAME test_persistence
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#define WAIT_TIME 60 /*seconds*/
+
+static const struct bt_mesh_pst_test_cfg save_cfg = {
+	.addr = 0x0123,
+	.dev_key = { 0xAA },
+	.net_idx = 0x0B,
+	.net_key = { 0xCC },
+	.flags = 0,
+	.iv_index = 0x1234,
```

Comment:
I still stand by my previous comment on this. These structures describe the nodes in the test, while the network is described in mesh_test.c, through its exposed `net_key` and `app_key` variables. Adding network information here just forces us to duplicate this info for each device in each test, which is unnecessary for all existing test cases. 

If you prefer not to add additional global variables, I would rather suggest moving the keys and parameters in the mesh_test.c file into a single struct, and name it `test_mesh` or something, then extern that in mesh_test.h